samples/presentation/create_by_providing_a_file_name/cs/Program.cs

```diff
@@ -1,294 +1,284 @@
-#nullable disable
 using DocumentFormat.OpenXml;
 using DocumentFormat.OpenXml.Drawing;
 using DocumentFormat.OpenXml.Packaging;
 using DocumentFormat.OpenXml.Presentation;
 using D = DocumentFormat.OpenXml.Drawing;
 using P = DocumentFormat.OpenXml.Presentation;
 
-namespace CreatePresentationDocument
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            string filepath = @"C:\Users\username\Documents\PresentationFromFilename.pptx";
-            CreatePresentation(filepath);
-        }
 
-        public static void CreatePresentation(string filepath)
-        {
-            // Create a presentation at a specified file path. The presentation document type is pptx, by default.
-            PresentationDocument presentationDoc = PresentationDocument.Create(filepath, PresentationDocumentType.Presentation);
-            PresentationPart presentationPart = presentationDoc.AddPresentationPart();
-            presentationPart.Presentation = new Presentation();
+CreatePresentation(args[0]);
 
-            CreatePresentationParts(presentationPart);
+static void CreatePresentation(string filepath)
+{
+    // Create a presentation at a specified file path. The presentation document type is pptx, by default.
+    PresentationDocument presentationDoc = PresentationDocument.Create(filepath, PresentationDocumentType.Presentation);
+    PresentationPart presentationPart = presentationDoc.AddPresentationPart();
+    presentationPart.Presentation = new Presentation();
 
-            //Dispose the presentation handle
-            presentationDoc.Dispose();
-        }
+    CreatePresentationParts(presentationPart);
 
-        private static void CreatePresentationParts(PresentationPart presentationPart)
```

Comment:
why is private removed?

Reply:
Because with private set I get [CS0106](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0106?f1url=%3FappId%3Droslyn%26k%3Dk(CS0106)): The modifier 'private' is not valid for this item" error from Visual Studio if is set to private.